### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by jefflill**
Added Ubuntu 20.04/xUbuntu_20.04 row to the environment variable table for Ubuntu.

https://github.com/cri-o/cri-o/issues/5838

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ To install on the following operating systems, set the environment variable $OS
 | Debian Testing   | `Debian_Testing`  |
 | Debian 10        | `Debian_10`       |
 | Raspberry Pi OS  | `Raspbian_10`     |
+| Ubuntu 22.04     | `xUbuntu_22.04`   |
 | Ubuntu 20.04     | `xUbuntu_20.04`   |
 | Ubuntu 19.10     | `xUbuntu_19.10`   |
 | Ubuntu 19.04     | `xUbuntu_19.04`   |
```
Comment (haircommander) [inline install.md:119]: would you mind pruning these and any other no longer supported distros

Comment (mythi) [reply install.md:119]: The list in the supported versions section should be updated too (or removed completely since it’s almost redundant info)

**Comment by TomSweeneyRedHat** [comment]
The addition LGTM, but I like the pruning idea.

File: install.md (hunk 1 above)
Comment (jefflill) [reply install.md:119]: Signed-off-by: Jeff Lill <jeff@lill.io>

**Comment by jefflill** [comment]
NOTE: **Podman v3.4.4**, not **v4.1.0** is being installed using the updated installation instructions with specifying:**OS=xUnuntu_22.04**

Is this the current intended official release?

**Comment by haircommander** [comment]
hey @jefflill can you sign-off your commit as described by our [contributors documentation](https://github.com/cri-o/cri-o/blob/main/CONTRIBUTING.md#sign-your-prs)

**Comment by haircommander** [comment]
sorry @jefflill all commits need to be signed-off, you can edit your original one with `git rebase -i HEAD~2` and selecting `r` on the first commit.

**Comment by saschagrunert** [comment]
A friendly reminder that this PR had no activity for 30 days.

**Pull request closed without merging**